public/js/angular/directives.js

```diff
@@ -81,7 +79,8 @@ angular
           });
         }
       };
-    }])
+    }
```

Comment:
this doesn't look like a fix that should be necessary, do you know which rule is triggering this? (if so, we might want to relax that in mofo-style)